### ISSUE TITLE
Configuration with environment variables does not update

### ISSUE THREAD

**Issue by wdullaer** [tenforce/docker-virtuoso#9]
There is a clause in the startup script which checks if there is an existing `virtuoso.ini` file. If this file exists the environment variables are not inserted into the ini file.
Because the `virtuoso.ini` file is saved in the same directory as the database files, it is very likely to live in a persistent volume.
This means that configuration variables are not updated when the environment variables change, even when the container is recreated (because the settings are persisted in the volume).

This is not how I would expect configuration variables to behave.

To resolve this we should probably move the `virtuoso.ini` file to a location outside the current database volume. This would ensure that the configuration is properly updated when recreating the container, while keeping the current existence check on `virtuoso.ini` in place to allow users to pass in a custom config file.
Users can still bind mount a different file to this new location is they chose not to use environment variables to configure virtuoso.

**Comment by parmentf** [comment]
Or, we could get rid of [the existence test for the configuration](https://github.com/tenforce/docker-virtuoso/blob/master/virtuoso.sh#L14), to update the `virtuoso.ini` using the environment variables at each start up.

**Comment by nvdk** [comment]
I agree that this seems to be unexpected behavior when creating a container. I'm not sure what most users want most: configuration using the ENV variables or live updating the config in the database. I'm afraid it's a bit of both and this creates some tension and limitations to the approaches we can take. 

I do think we only want to change the config only once and not on every start up as you suggest @parmentf. If a user updates the configuration using the conductor (or even manually) it should not be reset on the next startup imo.

The solution @wdullaer proposes would work, but may be confusing for users. The default for virtuoso is to have the virtuoso.ini file in the database directory and this is where most virtuoso users will look for it. If we move the file to another location it would have to be documented very clearly. 

We will also need to make sure any existing scripts take it into account by passing the +config property correctly to virtuoso-t. For example the default restore command would change.

**Comment by nvdk** [comment]
@madnificent pointed out a clean solution which would satisfy the paradigm "set config on (re)create but keep it on restart". We could move the .config_set file to a container directory. 

I believe we should do the same for .dba_pwd_set, for which I'll create a separate issue.
